internal/authorization/auth_model.go

```diff
@@ -0,0 +1,5 @@
+package authorization
+
+// Code generated by Makefile; DO NOT EDIT.
+
+var AuthModel = `{"schema_version":"1.1","type_definitions":[{"metadata":{"relations":{"accessible":{"directly_related_user_types":[{"type":"group","wildcard":{}},{"type":"role","wildcard":{}},{"type":"api","wildcard":{}}]}}},"relations":{"accessible":{"this":{}}},"type":"admin"},{"type":"user"},{"metadata":{"relations":{"assignee":{"directly_related_user_types":[{"type":"user"},{"relation":"member","type":"group"}]}}},"relations":{"assignee":{"this":{}}},"type":"role"},{"metadata":{"relations":{"member":{"directly_related_user_types":[{"type":"user"},{"relation":"member","type":"group"}]}}},"relations":{"member":{"this":{}}},"type":"group"},{"metadata":{"relations":{"admin":{"directly_related_user_types":[{"relation":"assignee","type":"role"},{"relation":"member","type":"group"}]},"can_create":{},"can_delete":{"directly_related_user_types":[{"type":"user"},{"relation":"assignee","type":"role"},{"relation":"member","type":"group"}]},"can_edit":{},"can_list":{},"can_view":{},"editor":{"directly_related_user_types":[{"type":"user"},{"relation":"assignee","type":"role"},{"relation":"member","type":"group"}]},"viewer":{"directly_related_user_types":[{"type":"user"},{"relation":"assignee","type":"role"},{"relation":"member","type":"group"}]}}},"relations":{"admin":{"this":{}},"can_create":{"union":{"child":[{"computedUserset":{"relation":"editor"}},{"computedUserset":{"relation":"admin"}}]}},"can_delete":{"union":{"child":[{"this":{}},{"computedUserset":{"relation":"admin"}}]}},"can_edit":{"union":{"child":[{"computedUserset":{"relation":"editor"}},{"computedUserset":{"relation":"admin"}}]}},"can_list":{"union":{"child":[{"computedUserset":{"relation":"viewer"}},{"computedUserset":{"relation":"admin"}}]}},"can_view":{"union":{"child":[{"computedUserset":{"relation":"viewer"}},{"computedUserset":{"relation":"admin"}}]}},"editor":{"this":{}},"viewer":{"union":{"child":[{"this":{}},{"computedUserset":{"relation":"editor"}}]}}},"type":"api"}]}`
```

Comment:
have a look at making it a yaml if possible

Reply:
ah I saw it in the other file, wondering if it s better to embed it and then call the function at bootstrap

Reply:
We use the fga CLI to transform it, we need something like this https://github.com/openfga/cli/blob/main/internal/authorizationmodel/model.go#L156. We can either install that module or copy/paste it, neither look very pretty.